civihr_default_theme/templates/page/page.tpl.php

```diff
@@ -48,16 +48,22 @@
       <div class="chr_header__nav__toggle">
         <i class="fa fa-2x fa-navicon"></i>
       </div>
-      <div class="chr_header__corner__brand chr_brand chr_header__home-menu" title="<?php print htmlspecialchars($site_name); ?>">
-          <span class="chr_brand__icon icon-logo"></span>
-          <span><?php print t("Home"); ?></span>
-        <ul class="chr_header__sub-menu">
-          <?php if (user_access("access CiviCRM")) { ?>
-            <li><?php print $admin_link; ?></li>
-          <?php } ?>
-          <li><?php print $ssp_link; ?></li>
-        </ul>
-      </div>
+      <?php if (!user_access("access CiviCRM")) { ?>
+        <a href="/dashboard">
+      <?php }?>
```

Comment:
I think it's better to use `endif` in these cases, makes the code more readable (http://php.net/manual/en/control-structures.alternative-syntax.php)